api/envoy/v9/http/path_matcher/config.proto

```diff
@@ -25,16 +25,6 @@ message PathMatcherRule {
 
   // Operation name (selector) used as a unique ID for the operation.
   string operation = 2 [(validate.rules).string.min_bytes = 1];
-
-  // If set, the path parameters in the `pattern` must be extracted. Ref:
-  // https://cloud.google.com/endpoints/docs/openapi/openapi-extensions#understanding_path_translation
-  PathParameterExtractionRule path_parameter_extraction = 3;
-}
-
-message PathParameterExtractionRule {
-  // Each rule has a mapping of snake_case segments to jsonCase segments.
-  // If the map is empty, the path params still need to be extracted.
-  map<string, string> snake_to_json_segments = 2;
 }
```

Comment:
We could not completely remove path_parameter_extraction.  Even it is an empty one, it indicates CONSTANT_ADDRESS and require to do variable names to query parameter translation,  
otherwise it is not needed.

We may use a bool to replace it.

Reply:
ACK